### PR TITLE
SC-6293 - adds learnstore permission

### PR DESCRIPTION
**Pull request by virgilchiriac**
# Description

adds a Learn-Store view permission, so that students are not given access to the Learn-Store

## Links to Tickets or other pull requests

- https://github.com/hpi-schul-cloud/schulcloud-server/pull/1671
- https://github.com/hpi-schul-cloud/nuxt-client/pull/1273
- https://ticketsystem.schul-cloud.org/browse/SC-6293

## Changes

removes the environmental variable check for the Learn-Store

## Approval for review
- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: helpers/handlebars/middleware.js [hunk 1]
```diff
@@ -104,13 +104,12 @@ module.exports = (req, res, next) => {
...
-			name: res.$t('global.sidebar.link.lernstore'),
-			icon: 'search',
-			link: '/content/',
-		});
-	}
+	res.locals.sidebarItems.push({
+		name: res.$t('global.sidebar.link.lernstore'),
+		icon: 'search',
+		link: '/content/',
+		permission: 'LERNSTORE_VIEW'
```
Comment (tofixx) [inline helpers/handlebars/middleware.js:111]:
```suggestion
		permission: 'LERNSTORE_VIEW',
```

**Review by tofixx — APPROVED** [review]
good

File: helpers/handlebars/middleware.js [hunk 2]
```diff
@@ -104,13 +104,12 @@ module.exports = (req, res, next) => {
 	}];
 
 	// Lern-Store Feature Toggle
```
Comment (Michael-Janke) [inline helpers/handlebars/middleware.js:106]: Die Zeile dann auch raus

**Comment by virgilchiriac** [comment]
obsolete - new PR to develop will follow

**Pull request closed without merging**